helm/cas-airflow/templates/routes/route.yaml

```diff
@@ -8,10 +8,10 @@ spec:
   host: {{ .Release.Name }}-{{ include "cas-airflow.namespaceSuffix" . }}.{{ .Values.route.clusterDomain }}
   to:
     kind: Service
-    name: {{ .Release.Name }}-webserver
+    name: {{ .Release.Name }}-api-server
     weight: 100
   port:
-    targetPort: airflow-ui
+    targetPort: api-server
```

Comment:
I think this will break access to the UI, unless the api server also serves the UI?

Reply:
We might need to expose 2 routes or maybe sub paths